backend/boards/api.py

```diff
@@ -48,9 +52,18 @@ def get_serializer_class(self):
     def get_queryset(self):
         user = self.request.user
         qs = super().get_queryset().filter(members=user)
-
+        assignees = self.request.query_params.get("assignees", None)
         if self.action == "retrieve":
-            return qs.prefetch_related("columns__tasks")
+            queryset = None
+            if assignees:
+                queryset = (
+                    Task.objects.filter(
+                        Q(assignees__in=[int(x) for x in assignees.split(",")])
```

Comment:
This works well, but as I mentioned in another comment - currently all of the state is already present in the redux store and thus the filtering could be done there.

Some ideas I've had for a better UX:
- Clicking on a member while holding down the cmd key filters tasks for that member and makes the highlights the avatar border
- Autocomplete like the one for choosing assignee in task card (doesn't make api calls and shows all users to choose from)
- The filter popover has list of user avatars, clicking on them will filter

Reply:
Thanks for the review. When I started to work on the project I started from the backend and had no idea that all the tasks are stored in the redux store. Considering that if the number of tasks might start to increase in a real project management environment and at some point it might not be possible to load all the tasks, don't you think that filtering on the backend can be useful? 

I guess the UX ideas are different suggestions?  Not all are needed? 
I copied the code for autocomplete from assignee in task, not sure why it is not loading. I will look into it.

Reply:
Yes, backend filtering is pretty much always a better approach since it's scalable. But to support many tasks e.g >10k some other things need to be done as well such as virtualized lists & boards with data only fetched for the visible viewport. This is not an easy task.

What makes sense to optimize currently is not fetch all the data for a task, but only the minimal amount to show the board view. Task descriptions could be fetched only when a task is opened (also having tasks visible in url would be nice as well). I'll make these into an issue at some point.

Yup the ideas were different suggestions.